BMM.UI.iOS/Application/Implementations/IosSetup.cs

```diff
@@ -80,6 +85,21 @@ protected override void InitializeFirstChance()
             RegisterMediaPlayer();
         }
 
+        public override void InitializeSecondary()
+        {
+            base.InitializeSecondary();
+            InitializeImageService();
+        }
+
+        private static void InitializeImageService()
+        {
+            ImageService.Instance.Initialize(new FFImageLoading.Config.Configuration
+            {
+                InvalidateLayout = false,
+                HttpClient = new HttpClient(new AuthenticatedNativeHttpImageClientHandler(Mvx.IoCProvider.Resolve<IMediaRequestHttpHeaders>()))
```

Comment:
doesn't this still use the same http client? I don't see how this uses the native http client